runners/katacoda/templates/createFile.md

```diff
@@ -0,0 +1,14 @@
+<%= text; %>
+
+<%= cdCommand; %>
+
+
+`mkdir -p <%= filePath; %>`{{execute}}
+
+Switch to the IDE and click 'Copy to Editor' in the following java class. The java file will be created automatically inside the newly created package.
```

Comment:
The file has not to be a java file in future. So we should keep this text more general. Maybe its better to specify the text in the playbook and pass the description by the text-placeholder into this file.